### PR TITLE
Virology SQL fix

### PR DESCRIPTION
**Pull request by kollil**
#### Rationale
Discovered a bug in the Virology billing review screen.
 The NIHrate field is empty for most of the records and that is not correct.
The NIHrate filed is coming a wrong dataset in the back end, I have corrected the query to display the accurate information.

### CODE REVIEW

**Review by jonesgaohsu — APPROVED** [review]

**Comment by kollil** [comment]
Gary, We already talked about it and you approved the changes in the afternoon. Did you find anything wrong after our meeting?. I can't merge until you approve it.

**Pull request merged**
